packages/cozy-stack-client/src/TriggerCollection.js

```diff
@@ -107,9 +107,9 @@ class TriggerCollection extends DocumentCollection {
    * @throws {FetchError}
    */
   async find(selector = {}, options = {}) {
-    if (Object.keys(selector).length === 1 && selector.worker) {
+    if (Object.keys(selector).length > 0 && selector.worker && selector.type) {
```

Comment:
This change is a breaking change, and I'm not sure we really want to force the use of the `type`. In some case, it is wanted to not select on the type. 

We should handle those cases: 
- only worker
- only type
- both of them 

WDYT?

Reply:
I agree. Moreover the stack implements a way to filter through multiple types (/jobs/triggers?Type=@cron,@webhook). Since Mango already supports this via $in, I propose that we support:

```
Q('io.cozy.triggers').where({
  type: {
    $in: ['@cron', '@webhook']
  }
})

Q('io.cozy.triggers').where({
  type: '@cron'
})

Q('io.cozy.triggers').where({
  type: '@webhook'
})
```

Reply:
cc @nono, are there other APIs from the stack that have the same filtering ability ? Would it make sense for those APIs to support the mango selector so that it is not done client-side ? It would make the client-side simpler.

Reply:
> are there other APIs from the stack that have the same filtering ability ?

I think we can find 2 or 3 endpoints with filtering like that. Scanning the full docs should answer that.

> Would it make sense for those APIs to support the mango selector so that it is not done client-side ?

Meh, I don't like that. It means creating a new endpoint with `POST`, and parsing the mango selectors, and making a response compatible with the expected format. And it is probably a good way to have silent errors (like putting a limit in the mango filter that will be ignored).

Reply:
I made changes taking into account your comments